controllers/repliesController/deleteSingleReply.js

```diff
@@ -51,17 +52,18 @@ const deleteSingleReply = async (req, res, next) => {
     if (!reply) {
       return next(new CustomError(404, "reply not found"));
     }
-    await Comments.findByIdAndUpdate(commentId, {
-      // it doesn't matter if the parent exist or not
-      $pull: { replies: replyId },
-    });
+    // await Comments.findByIdAndUpdate(commentId, {
+    //   // it doesn't matter if the parent exist or not
+    //   $pull: { replies: replyId },
```

Comment:
please do not comment out, I think this updates the `comment.replies` array in the db